Topic 1/src/com/globant/topic1/exercise1/model/Cart.java

```diff
@@ -0,0 +1,46 @@
+package com.globant.topic1.exercise1.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.globant.topic1.exercise1.strategy.IPaymentMethodStrategy;
+
+public class Cart {
+
+	// Fields
+	private List<Item> items;
+
+	// Constructor
+	public Cart() {
+		this.items = new ArrayList<>();
+	}
+
+	public void addItem(Item newItem) {
+		// If item exist, add the new quantity
+		if (items.contains(newItem)) {
+			int pos = items.indexOf(newItem);
+			Item oldItem = items.get(pos);
+			oldItem.setQuantity(oldItem.getQuantity() + newItem.getQuantity());
+		} else {
+			this.items.add(newItem);
+		}
+	}
+
+	public void removeItem(Item item) {
+		this.items.remove(item);
```

Comment:
Advice: you can check if the size of the list is greater than 1, if not, the remove won't be necesary.